ansible/roles/test/tasks/snmp/interfaces.yml

```diff
@@ -5,13 +5,22 @@
 
 - set_fact:
     snmp_intf: []
+    snmp_mgmt_intf: []
     mg_intf: []
 
+# Gather front panel ports and port channels
 - name: Create snmp interfaces list
   set_fact: 
     snmp_intf: "{{ snmp_intf + [item.value.name] }}"
   with_dict: "{{ snmp_interfaces }}"
-  when: "{{item.value.name is defined}}"  
+  when: "{{item.value.name is defined and item.value.ifindex | int < 9999 }}"
```

Comment:
>int < 9999 [](start = 63, length = 10)

This is a snmpagent implementation detail and the outside should never depend on it.